### ISSUE TITLE
Bug...

### ISSUE THREAD

**Issue by Kumaru01** [jsmsj/GdriveCloneWeb#2]
You need to create dir 'tmp' before assign it to filehandler for logger.

**Comment by DELL150150** [comment]
> You need to create dir 'tmp' before assign it to filehandler for logger.

where do i need to make it in phone(termux) and pc ,i tried mkdir tmp
i still get that error

**Comment by stuffed18** [comment]
Yup, if you run your script you will get one error from the logger python script, and one error for the main.py script, because directory tmp does not exist. I'm not sure about where u would do it on linux, but for windows just create a tmp folder in in C: drive (so it'll be C:\tmp). You could modify the script to solve this without too much difficulty, I just changed the directory to a more convenient location.
